src/interface.ts

```diff
@@ -0,0 +1,27 @@
+import React from 'react';
```

Comment:
In practice, creating files or directories named `types...` or `interfaces...` doesn't scale well. If it's small and well contained within the package it's used it should be fine, but at `src` it's bound to be an early candidate for cleanup

Reply:
So for scaling, would it just be best to add the interfaces/types to whichever file they're being used in?

Reply:
Kind of, my philosophy is to define things close to where they're used

Reply:
Got that refactored in my last commit